shaman/models/builds.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime
+from sqlalchemy.orm import relationship, backref, deferred
+from sqlalchemy.event import listen
+from sqlalchemy.orm.exc import DetachedInstanceError
+from shaman.models import Base, update_timestamp
+from shaman.models.types import JSONType
+
+
+class Build(Base):
+
+    __tablename__ = 'builds'
+    id = Column(Integer, primary_key=True)
+    url = Column(String(256))
```

Comment:
hrmn, this `build_url` was initially `log_url`. 

`url` was meant a `url` pointing back to the job url (not the log)